be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/LaoValidator.scala

```diff
@@ -0,0 +1,78 @@
+package ch.epfl.pop.pubsub.graph.validators
+
+import ch.epfl.pop.model.network.JsonRpcRequest
+import ch.epfl.pop.model.network.method.message.Message
+import ch.epfl.pop.model.network.method.message.data.lao.{CreateLao, StateLao, UpdateLao}
+import ch.epfl.pop.model.objects.Hash
+import ch.epfl.pop.pubsub.graph.{GraphMessage, PipelineError}
+
+case object LaoValidator extends ContentValidator {
+  sealed def validateCreateLao(rpcMessage: JsonRpcRequest): GraphMessage = {
+    def validationError(reason: String): PipelineError = super.validationError(reason, "CreateLao")
```

Comment:
in these validators, how does one know which JSON-RPC id / message_id this refers to ? 
I assume we'll want to log this stuff, and it's important to be able to trace it back to a specific request.

Reply:
I think this issue will be resolved if I manage to implement the UserActors we were talking about

Reply:
I'm not sure about this :)